src/app/extensions/status-cell-renderer/status-cell-renderer.component.ts

```diff
@@ -39,6 +40,8 @@ export class StatusCellRendererComponent implements ICellRendererAngularComp {
   released: boolean;
   releaseText: string;
 
+  monthYearRegExp: RegExp = /^([a-zA-Z]{3})\s+(20\d{2}|\d{2})$/;
```

Comment:
This is not very tight, and will catch non date strings.

Reply:
Fixed it.